### ISSUE TITLE
Feature request - Manual override - CAN contactors

### ISSUE THREAD

**Issue by frasersimpson83** [dalathegreat/Battery-Emulator#404]
Would it be possible to add a manual override for batteries with CAN based contactor controls. 

This is so that if we want to open contactors but leave everything else in place we can.   Currently for Tesla Model 3 batteries for instance we need to remove 12V connection to the BMS and contactors to open them which is fine, but if remote from the battery it would be nice to be able to open the contactors and disconnect the battery manually.

This setting should stay active even during a restart for safety purposes and then the user has to connect to the web interface and put it back into 'normal operation' where the contactors would close automatically.

This would be extremely helpful in multiple situations and I'm guessing not that hard to implement since we already have all the code there, it's just adding a manual over-ride switch somewhere.

**Comment by dalathegreat** [comment]
Afaik adding a button would not help for Tesla, since we have no known way to open the contactors once the pack is going.

**Comment by frasersimpson83** [comment]
But when the Lilygo errors out for whatever reason and it goes red the contactors open...... maybe it's the Tesla BMS opening the contactors or something but I assumed it was the Lilygo sending something to try to open them ?

**Comment by dalathegreat** [comment]
@frasersimpson83 Yes, this is the battery itself taking that action. We do not know which CAN messages are needed to open the contactors.

**Comment by dalathegreat** [comment]
Closing as completed I think!
